.github/workflows/extension.yml

```diff
@@ -76,9 +76,9 @@ jobs:
 
       - name: Update version numbers in task.json
         run: |
-          echo "`jq '.version.Major=${{ steps.gitversion.outputs.major }}' extension/tasks/dependabot/dependabotV1/task.json`" > extension/tasks/dependabot/dependabotV1/task.json
-          echo "`jq '.version.Minor=${{ steps.gitversion.outputs.minor }}' extension/tasks/dependabot/dependabotV1/task.json`" > extension/tasks/dependabot/dependabotV1/task.json
-          echo "`jq '.version.Patch=${{ github.run_number }}' extension/tasks/dependabot/dependabotV1/task.json`" > extension/tasks/dependabot/dependabotV1/task.json
+          echo "`jq '.version.Major=${{ steps.gitversion.outputs.major }}' extension/tasks/dependabot/dependabotV2/task.json`" > extension/tasks/dependabot/dependabotV2/task.json
+          echo "`jq '.version.Minor=${{ steps.gitversion.outputs.minor }}' extension/tasks/dependabot/dependabotV2/task.json`" > extension/tasks/dependabot/dependabotV2/task.json
+          echo "`jq '.version.Patch=${{ github.run_number }}' extension/tasks/dependabot/dependabotV2/task.json`" > extension/tasks/dependabot/dependabotV2/task.json
```

Comment:
Yeh, I wasn't sure how this was going to work to be honest. Should I make it use to the same version number in both files? Will DevOps get confused by two tasks with the same [major] version?

My thinking was to make V2's version automatically update as normal, but V1 would be manually updated only if there is a patch/hot-fix to that script.

Let me know how you see this working and I'll update it

Reply:
I see what you mean about needing to bump the patch; I've set it to update V1 patch but kept the major/minor static to whatever the "last 1.x release" was. 

If you want the V1 minor to be incremented, what should the value be?
Playing a few scenarios out in my head, using `gitversion.minor` might cause regressions if you created a "2.1.0" release, for example. To keep it from regressing, it would need to always be a minimum 34 and always higher than the last version number. I could be overthinking this though, let me know if i'm missing something obvious here.

Reply:
You have a very valid view here.
We can assume we do not need other make any more changes to it.